{
   "active_lock_reason" : null,
   "assignee" : null,
   "assignees" : [],
   "author_association" : "CONTRIBUTOR",
   "body" : "Fixes #23990\r\n\r\nAfter #23799, the `--failfast` option in the test runner for the functional tests stopped working, because a second outer loop was introduced, which would have needed a `break` too for the test runner to fail immediately. This also led to the errors reported in #23990.\r\n\r\nThis provides a straightforward fix for that.\r\nThere is also #23995 which is a larger refactor, but that hasn't been updated in a while to fix the failfast issue.",
   "closed_at" : null,
   "closed_by" : null,
   "comments" : 0,
   "comments_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/24195/comments",
   "created_at" : "2022-01-28T14:56:45Z",
   "draft" : false,
   "events_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/24195/events",
   "html_url" : "https://github.com/bitcoin/bitcoin/pull/24195",
   "id" : 1117499192,
   "labels" : [],
   "labels_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/24195/labels{/name}",
   "locked" : false,
   "milestone" : null,
   "node_id" : "PR_kwDOABII584xvepj",
   "number" : 24195,
   "performed_via_github_app" : null,
   "pull_request" : {
      "diff_url" : "https://github.com/bitcoin/bitcoin/pull/24195.diff",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/24195",
      "merged_at" : null,
      "patch_url" : "https://github.com/bitcoin/bitcoin/pull/24195.patch",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/pulls/24195"
   },
   "reactions" : {
      "+1" : 0,
      "-1" : 0,
      "confused" : 0,
      "eyes" : 0,
      "heart" : 0,
      "hooray" : 0,
      "laugh" : 0,
      "rocket" : 0,
      "total_count" : 0,
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/24195/reactions"
   },
   "repository_url" : "https://api.github.com/repos/bitcoin/bitcoin",
   "state" : "open",
   "timeline_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/24195/timeline",
   "title" : "test: Fix failfast option for functional test runner",
   "updated_at" : "2022-01-28T14:56:45Z",
   "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/24195",
   "user" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/48763452?v=4",
      "events_url" : "https://api.github.com/users/mzumsande/events{/privacy}",
      "followers_url" : "https://api.github.com/users/mzumsande/followers",
      "following_url" : "https://api.github.com/users/mzumsande/following{/other_user}",
      "gists_url" : "https://api.github.com/users/mzumsande/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/mzumsande",
      "id" : 48763452,
      "login" : "mzumsande",
      "node_id" : "MDQ6VXNlcjQ4NzYzNDUy",
      "organizations_url" : "https://api.github.com/users/mzumsande/orgs",
      "received_events_url" : "https://api.github.com/users/mzumsande/received_events",
      "repos_url" : "https://api.github.com/users/mzumsande/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/mzumsande/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/mzumsande/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/mzumsande"
   }
}
